tools/run-qemu.py

```diff
@@ -21,6 +22,9 @@
             "-device", "virtio-net,netdev=net0,disable-legacy=on,disable-modern=off",
             "-netdev", "user,id=net0,hostfwd=tcp:127.0.0.1:20022-:22,hostfwd=tcp:127.0.0.1:20080-:80",
             "-object", "filter-dump,id=fiter0,netdev=net0,file=virtio-net.pcap",
+            
+            "-device", "virtio-blk-pci,drive=drive0,id=virtblk0,disable-legacy=on,disable-modern=off",
+            "-drive", "file=vdisk,if=none,id=drive0",
```

Comment:
Tip: QEMU has a virtual FAT formatted disk and you may find it convenient
https://en.wikibooks.org/wiki/QEMU/Devices/Storage

Reply:
hey @nuta. I'm confused as to what exact qemu argument I should pass. The latest commit to ```run-qemu.py``` exits to status 0. I've also tried the following:
```python
"-drive", "file=vdisk,if=none,format=raw,id=hd",
"-device", "virtio-blk-pci,drive=hd",
```
The strings above compile however the ```virtioblock::new()`` method isn't being instantiated. Help would be much appreciated.

Reply:
`disable-legacy=on,disable-modern=off` is essential because the current virtio implementation supports the virtio modern spec only.

```
            "-drive", "file=vdisk,if=none,format=raw,id=hd",
            "-device", "virtio-blk-pci,drive=hd,disable-legacy=on,disable-modern=off",
```

Reply:
I see thanks, in regards to the qemu  failure status I think it's likely the ```request_to_device()``` method that isn't well fleshed out. I'll be looking over the virtio spec some more and hopefully have the driver ready to merge by the end of the weekend.

Reply:
Sorry for the late reply. I  just pushed the latest commit and addressed merge conflicts. I am still getting kernel panics without messages. Haven't been able to figure out why yet.